src/main.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
By removing the boilerplate I mentioned also that you need to create a sample preview not remove the whole file content!